MoreLinq/Repeat.cs

```diff
@@ -38,8 +38,7 @@ public static IEnumerable<T> Repeat<T>(this IEnumerable<T> sequence, int count)
         }
 
         /// <summary>
-        /// Repeats the specific sequences indefinitely, or equivalently, creates a circular sequence
-        /// from the original source sequence.
+        /// Repeats the specific sequences forever.
```

Comment:
I know that the original `Repeat` summary was poor, but perhaps we can revise it for the two overloads now? That is, _sequences_ should be _sequence_ (there is only one). And do we need to say _specific_? Simplify to just:

> /// Repeats the sequence forever.